😢  It appears the `available_languages.xml` issue was [not the only thing that made the Jetpack Lint fail](https://buildkite.com/automattic/wordpress-android/builds/6047#01826fe3-a404-4a20-a32c-b60d1daa21cb/137-212):

```
[2022-08-05T21:26:33Z] FAILURE: Build failed with an exception.
--
  | [2022-08-05T21:26:33Z]
  | [2022-08-05T21:26:33Z] * What went wrong:
  | [2022-08-05T21:26:33Z] Execution failed for task ':WordPress:processJetpackVanillaReleaseGoogleServices'.
  | [2022-08-05T21:26:33Z] > No matching client found for package name 'com.jetpack.android'

```

I'll need to investigate that further next week before being ready to un-draft this PR.

--- a/.buildkite/beta-builds.yml
+++ b/.buildkite/beta-builds.yml
@@ -15,11 +15,11 @@ steps:
     artifact_paths:
       - "**/build/reports/lint-results*.*"
 
-  # - label: "🔬 Lint Jetpack"
-  #   command: ".buildkite/commands/lint.sh jetpack"
-  #   key: jplint
-  #   artifact_paths:
-  #     - "**/build/reports/lint-results*.*"
+  - label: "🔬 Lint Jetpack"
+    command: ".buildkite/commands/lint.sh jetpack"
+    key: jplint
+    artifact_paths:
+      - "**/build/reports/lint-results*.*"
 
   - label: "🛠 WordPress Beta Build"
     command: ".buildkite/commands/beta-build.sh wordpress"
@@ -30,7 +30,7 @@ steps:
 
   - label: "🛠 Jetpack Beta Build"
     command: ".buildkite/commands/beta-build.sh jetpack"
-    # depends_on: jplint
+    depends_on: jplint
     plugins: *common_plugins
     notify:
       - slack: "#build-and-ship"

--- a/.buildkite/pipeline.yml
+++ b/.buildkite/pipeline.yml
@@ -31,10 +31,10 @@ steps:
         artifact_paths:
           - "**/build/reports/lint-results*.*"
 
-      # - label: "Lint Jetpack"
-      #   command: ".buildkite/commands/lint.sh jetpack"
-      #   artifact_paths:
-      #     - "**/build/reports/lint-results*.*"
+      - label: "Lint Jetpack"
+        command: ".buildkite/commands/lint.sh jetpack"
+        artifact_paths:
+          - "**/build/reports/lint-results*.*"
 
   - label: "Dependency Tree Diff"
     command: |

--- a/.buildkite/release-builds.yml
+++ b/.buildkite/release-builds.yml
@@ -15,11 +15,11 @@ steps:
     artifact_paths:
       - "**/build/reports/lint-results*.*"
 
-  # - label: "🔬 Lint Jetpack"
-  #   command: ".buildkite/commands/lint.sh jetpack"
-  #   key: jplint
-  #   artifact_paths:
-  #     - "**/build/reports/lint-results*.*"
+  - label: "🔬 Lint Jetpack"
+    command: ".buildkite/commands/lint.sh jetpack"
+    key: jplint
+    artifact_paths:
+      - "**/build/reports/lint-results*.*"
 
   - label: "🛠 WordPress Release Build"
     command: ".buildkite/commands/release-build.sh wordpress"
@@ -30,7 +30,7 @@ steps:
 
   - label: "🛠 Jetpack Release Build"
     command: ".buildkite/commands/release-build.sh jetpack"
-    # depends_on: jplint
+    depends_on: jplint
     plugins: *common_plugins
     notify:
       - slack: "#build-and-ship"

--- a/WordPress/google-services.json-example
+++ b/WordPress/google-services.json-example
@@ -129,6 +129,132 @@
           "status": 1
         }
       }
+    },
+    {
+      "client_info": {
+        "mobilesdk_app_id": "1:123:android:abc",
+        "android_client_info": {
+          "package_name": "com.jetpack.android"
+        }
+      },
+      "oauth_client": [
+        {
+          "client_id": "123-abc.apps.googleusercontent.com",
+          "client_type": 3
+        },
+        {
+          "client_id": "",
+          "client_type": 1,
+          "android_info": {
+            "package_name": "com.jetpack.android",
+            "certificate_hash": ""
+          }
+        }
+      ],
+      "api_key": [
+        {
+          "current_key": ""
+        }
+      ],
+      "services": {
+        "analytics_service": {
+          "status": 2,
+          "analytics_property": {
+            "tracking_id": ""
+          }
+        },
+        "appinvite_service": {
+          "status": 1,
+          "other_platform_oauth_client": []
+        },
+        "ads_service": {
+          "status": 1
+        }
+      }
+    },
+    {
+      "client_info": {
+        "mobilesdk_app_id": "1:123:android:abc",
+        "android_client_info": {
+          "package_name": "com.jetpack.android.prealpha"
+        }
+      },
+      "oauth_client": [
+        {
+          "client_id": "123-abc.apps.googleusercontent.com",
+          "client_type": 3
+        },
+        {
+          "client_id": "",
+          "client_type": 1,
+          "android_info": {
+            "package_name": "com.jetpack.android.prealpha",
+            "certificate_hash": ""
+          }
+        }
+      ],
+      "api_key": [
+        {
+          "current_key": ""
+        }
+      ],
+      "services": {
+        "analytics_service": {
+          "status": 2,
+          "analytics_property": {
+            "tracking_id": ""
+          }
+        },
+        "appinvite_service": {
+          "status": 1,
+          "other_platform_oauth_client": []
+        },
+        "ads_service": {
+          "status": 1
+        }
+      }
+    },
+    {
+      "client_info": {
+        "mobilesdk_app_id": "1:123:android:abc",
+        "android_client_info": {
+          "package_name": "com.jetpack.android.beta"
+        }
+      },
+      "oauth_client": [
+        {
+          "client_id": "123-abc.apps.googleusercontent.com",
+          "client_type": 3
+        },
+        {
+          "client_id": "",
+          "client_type": 1,
+          "android_info": {
+            "package_name": "com.jetpack.android.beta",
+            "certificate_hash": ""
+          }
+        }
+      ],
+      "api_key": [
+        {
+          "current_key": ""
+        }
+      ],
+      "services": {
+        "analytics_service": {
+          "status": 2,
+          "analytics_property": {
+            "tracking_id": ""
+          }
+        },
+        "appinvite_service": {
+          "status": 1,
+          "other_platform_oauth_client": []
+        },
+        "ads_service": {
+          "status": 1
+        }
+      }
     }
   ],
   "configuration_version": "1"

--- a/WordPress/src/jetpack/res/values/available_languages.xml
+++ b/WordPress/src/jetpack/res/values/available_languages.xml
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--Warning: Auto-generated file, do not edit.-->
-<resources>
-  <string-array name="available_languages" translatable="false">
+<resources xmlns:tools="http://schemas.android.com/tools">
+  <string-array name="available_languages" translatable="false" tools:ignore="InconsistentArrays">
     <item>en_US</item>
     <item>ar</item>
     <item>de</item>

--- a/WordPress/src/main/res/values/available_languages.xml
+++ b/WordPress/src/main/res/values/available_languages.xml
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--Warning: Auto-generated file, do not edit.-->
-<resources>
-  <string-array name="available_languages" translatable="false">
+<resources xmlns:tools="http://schemas.android.com/tools">
+  <string-array name="available_languages" translatable="false" tools:ignore="InconsistentArrays">
     <item>en_US</item>
     <item>ar</item>
     <item>de</item>

